src/main/java/com/banking/money/transaction/TransactionDaoImpl.java

```diff
@@ -13,7 +13,7 @@
 import com.banking.exceptions.TransactionDBAccessException;
 
 /**
- * @author Dipanjan Sengupta
+ * @author Dipanjan Sengupta 
  * @purpose - HibernateTemplate for operations on transactions table
  */
 public class TransactionDaoImpl implements TransactionDao{
```

Comment:
- hibernate template shouldn't have a getter method since it is dependency injected
- log all TransactionException after catching them and before wrapping into custom exception
- readOnly property of Transactional annotation for get all transactions, get transaction details should be true
- the method signature for getting transaction details should contain the parameter names in the method name